src/components/SubmissionForm/ProjectDetails.js

```diff
@@ -160,14 +160,31 @@ const ProjectDetails = props => {
           onChange={e => handleStateChanges(e)}
         />
         <br />
-        <Button
-          label="Continue"
-          type="submit"
-          color="primary"
-          onClick={e => Continue(e)}
-        >
-          Continue
-        </Button>
+        {name === "" ||
+        description === "" ||
+        hosted_url === "" ||
+        frontend_url === "" ||
+        backend_url === "" ||
+        display_image === "" ? (
```

Comment:
This can be made cleaner by using the not`!` operator to check if null, instead of `property === ""`

Reply:
``` !name || !description || !hosted_url || !frontend_url || !backend_url || !display_image ? *disable button* : *enable button* ```

Reply:
Fixed!